status.sh

```diff
@@ -4,4 +4,3 @@ echo "STABLE_MAIN_VERSION $(make --quiet --no-print-directory tag)"
 echo "STABLE_COLLECTOR_VERSION $(cat COLLECTOR_VERSION)"
 echo "STABLE_SCANNER_VERSION $(cat SCANNER_VERSION)"
 echo "STABLE_GIT_SHORT_SHA $(make --quiet --no-print-directory shortcommit)"
-echo "BUILD_TIMESTAMP $(date '+%s')"
```

Comment:
1. Please add entry for 4.0 in the CHANGELOG.md saying that this was removed.
2. Please update the existing entry in 3.74 in CHANGELOG.md to say "4.0" instead of "3.75".
3. Could you please not remove the default template of PR description for future PRs and restore it in this PR? It mentions checklist and testing done. There were good reasons to introduce that, I suppose. One of those good reasons is a reminder about the changelog.

Reply:
1. Tests were edited
2. No upgrade steps 
3. No documentation needed